src/deferred.js

```diff
@@ -359,5 +363,22 @@ jQuery.extend( {
 	}
 } );
 
+if ( window.console && window.console.warn ) {
+
+	// These usually indicate a programmer mistake during development,
+	// warn about them ASAP rather than swallowing them by default.
+
+	var rerrorNames = /^(Eval|Internal|Range|Reference|Syntax|Type|URI)Error$/;
+
+	jQuery.Deferred.exceptionHook = function( error ) {
+		if ( error && rerrorNames.test( error.name ) ) {
+			window.console.warn( "jQuery.Deferred exception: " + error.message );
```

Comment:
Ah, OK. I *think* if you open the site with DevTools open and then close it the `console` object is still there but I'd have to check.